cli/default-reporter/src/reportError.ts

```diff
@@ -361,7 +361,6 @@ function reportAuthError (
   const foundSettings = [] as string[]
   for (const [key, value] of Object.entries(config?.rawConfig ?? {})) {
     if (key.startsWith('@')) {
-      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
The `@typescript-eslint/restrict-template-expressions` rule relaxed what types are allowed in template expressions.

https://github.com/typescript-eslint/typescript-eslint/commit/c13ce0b4f7a74a6d8fecf78d25ebd8181f7a9119#diff-b852e1e199d2976eee1183fc84ac12a5d42fc61f0ae4b1c290dd54d621546db0

Many of these disables were for interpolated values that had an `any` type. Instead of removing these disable lines, we could also set the values changed in the commit above (e.g. `allowAny`) back to `false`.